src/main/java/mat/server/service/impl/MatContextServiceUtil.java

```diff
@@ -184,17 +185,32 @@ public boolean isCurrentCQLLibraryDraftable(CQLLibraryDAO cqlLibraryDAO,
         return isCurrentCQLLibraryEditable(cqlLibraryDAO, libraryId, false);
     }
 
+    public boolean isValidatable(Measure measure) {
+        return isFhirMeasureValidatabe(measure) || isQdmMeasureValidatable(measure);
+    }
+
+    private boolean isQdmMeasureValidatable(Measure measure) {
+        return !measure.isDraft() &&
+                measure.isQdmMeasure() &&
+                isVersionEligibleForFhirValidation(measure.getQdmVersion(), measure.getReleaseVersion()) &&
+                !BooleanUtils.isTrue(measure.getIsCompositeMeasure());

```

Comment:
Depending on how you feel about which is easier to read, BooleanUtils also has a isFalse() method (though I have not verified it is in the version of BooleanUtils we are using).

Reply:
BooleanUtils.isFalse is checking specifically for False value. which means null will be treated as True

Reply:
isFalse:
`Checks if a Boolean value is false, handling null by returning false.`.
From [javadocs](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/BooleanUtils.html#isFalse-java.lang.Boolean-)

Reply:
public static boolean isFalse(Boolean bool) {
        return Boolean.FALSE.equals(bool);
    }

Reply:
Ah, got it. You suggested isNotTrue

Reply:
Did you want null to return true? I was assuming if measure.getIsCompositeMeasure() was null, then it wasn't a composite measure and should be considered false.